ocs_ci/deployment/deployment.py

```diff
@@ -309,6 +294,32 @@ def do_gitops_deploy(self):
             gitops_obj._has_phase = True
             gitops_obj.wait_for_phase("successful", timeout=720)
 
+    def get_clusterset_name(self):
+        """
+        Function to fetch unique cluster set name used by managed clusters
+
+        Returns: list of cluster set
+
+        """
+
+        cluster_set = []
+        managed_clusters = (
+            ocp.OCP(kind=constants.ACM_MANAGEDCLUSTER).get().get("items", [])
+        )
+        # ignore local-cluster here
+        for i in managed_clusters:
+            if i["metadata"]["name"] != constants.ACM_LOCAL_CLUSTER:
+                cluster_set.append(
+                    i["metadata"]["labels"][constants.ACM_CLUSTERSET_LABEL]
+                )
+        if all(x == cluster_set[0] for x in cluster_set):
+            logger.info(f"Found the uniq clusterset {cluster_set[0]}")
+        else:
+            raise UnexpectedDeploymentConfiguration(
+                "There are more then one clusterset added to multiple managedcluters"
+            )
+        return cluster_set
+
```

Comment:
Better to have in utility file

Reply:
which utility file?

Reply:
It makes more sense to have it where it is because of it's usage and other relevant functions

Reply:
Not a good approach to call function for here to diff function

Reply:
ok, updated!